### PR TITLE
Custom region model trace aggregation

### LINKED ISSUE #426: Generate provincial forecasts for Pakistan
We have forecasts for ~30 different regions, but we want to turn a weighted linear combination of the traces into forecasts for each of the 7 distinct administrative units. 

Connor has provided the relevant weights here, in the "Region to Province map": https://docs.google.com/spreadsheets/d/1IxPMadPxjnphWSKG_6PxmsrCLoXe3cHGp1Ok9kcddPk/edit#gid=1378327731 (you will need to be granted access)

Now we just need some script for making the aggregation. If it could be added to epimodel, in reused in future when modelling novel provinces in countries, that could be great. 

Basically, the output of the script should be an additional set of JSON-files in the web-export, labelled "extdata-[add PROVINCE_NAME].json" but where the infected/recovered/active numbers are just a weighted sum of the numbers for the relevant regions. (And the "Statistics" should be recomputed for the new traces.)

### PR DESCRIPTION
**Pull request by wolverdude**
Resolves [Covid#426](https://github.com/epidemics/covid/issues/426).

This works by adding another type of regions file that can be fed into `RegionDataset`. The default file for that is being added in [eipimodel-covid-data#12](https://github.com/epidemics/epimodel-covid-data/pull/12), which must be merged before this PR.

### CODE REVIEW

File: epimodel/exports/epidemics_org.py [hunk 1]
```diff
@@ -437,6 +437,44 @@ def aggregate_countries(
...
+        # compute aggregate weights
+        child_codes = [child.Code for child in reg.children]
+        weights = pd.Series(
+            [
+                reg.model_weights[child.Code] * child.Population
+                for child in reg.children
+            ],
+            index=child_codes,
+        )
+        weights /= weights.sum()
```
Comment (wolverdude) [inline epimodel/exports/epidemics_org.py]: @lagerros How is this?

**Comment by wolverdude** [comment]
I should note that my implementation creates an implicit order dependency which isn't a problem at the moment because custom regions are loaded last, but if that changed or if someone created a custom region that depends on another custom region, it could break.

The correct solution to this is the whole dependency graph thing, but I figured let's wait until we implement regions in general as a graph structure before doing this.

**Review by mathijshenquet — APPROVED** [review]
Looks good to me

**Comment by mathijshenquet** [comment]
@wolverdude can we merge this? The follow up branch https://github.com/epidemics/epimodel/pull/59 is used to build production

**Pull request merged**